erpnext/setup/doctype/employee/employee.js

```diff
@@ -83,6 +83,7 @@ frappe.ui.form.on("Employee", {
 			},
 			callback: function (r) {
 				frm.set_value("user_id", r.message);
+				frm.save();
```

Comment:
Better way would be to set it directly in db and update the form. Thoughts?

Reply:
@rtdany10 Can you implement and push in the same PR?

Reply:
Updated